docs/content/dev/getting-started.md

```diff
@@ -1,8 +1,41 @@
 # Getting started
 
+This document will help you with setting up the development environment for HedgeDoc 2.
+
 ## Preparation
 
-Setup the [backend](./setup/backend.md) and the [frontend](./setup/frontend.md).
+### Preparing the directory
+
+1. Make sure that NodeJS is installed. You need at least Node 14 (we recommend Node 18).
+2. Make sure that [yarn](https://yarnpkg.com/) is installed.
+3. Clone this repo (e.g. `git clone https://github.com/hedgedoc/hedgedoc.git hedgedoc`)
+4. Go into the cloned directory
+
+### Installing the dependencies
+
+1. Go into the root directory of the cloned repository.
```

Comment:
I think this is same as in 4. in preparing directory.

Reply:
Rephrased